ci/requirements-3.6-dev.yml

```diff
@@ -1,7 +1,7 @@
 channels:
   - conda-forge
 dependencies:
-  - black
+  - git+git://github.com/psf/black@40e8b3a231bade22d838858435a33d60a8325306#egg=black  # see .pre-commit-config.yaml, we pin to a specific version of black
```

Comment:
this probably should be in a pip section.

at the end, inside the pip section after ` - seed-isort-config` you can add:

```
    - git+https://github.com/psf/black.git@40e8b3a231bade22d838858435a33d60a8325306
    - git+https://github.com/timothycrosley/isort.git@18ad293fc9d1852776afe35015a932b68d26fb14
```

also we need a specific version of isort

Reply:
Thanks! I should have seen the pip section before...